Eplant/EplantLayout.tsx

```diff
@@ -29,15 +40,19 @@ const EplantLayout = () => {
     }
   }, [theme, loaded])
 
+  //Initializing isCollapse variable to isCollapse jotai atom state
+  const [isCollapse] = useAtom(isCollapseState)
+
   return (
     <Box
       sx={(theme) => ({
-        height: `100%`,
-        left: `${sidebarWidth}px`,
+        height: `calc(100% - ${theme.spacing(1)})`,
+        left: `${isCollapse ? 100 : sidebarWidth}px`,
```

Comment:
Nit: Can you define another variable called collapsedSidebarWidth and put it in `Eplant/UI/Sidebar.tsx`